src/components/AuthorLink/AuthorLink.tsx

```diff
@@ -0,0 +1,37 @@
+import { useTranslation } from 'react-i18next';
+import GitHubIcon from '@mui/icons-material/GitHub';
+import { Button, Link } from '@mui/material';
+
+interface IAuthorLink {
+  name: string;
+  url: string;
+}
+
+const AuthorLink = ({ name, url }: IAuthorLink): JSX.Element => {
+  const { t } = useTranslation();
+
+  return (
+    <Button
+      variant="text"
+      component={Link}
+      href={url}
+      target="_blank"
+      rel="noreferrer"
+      sx={{
+        '&:hover': {
+          color: 'primary.dark',
```

Comment:
I would created some file in component's folder like 'styles.js' and hold this styles inside it